src/main/groovy/edu/oregonstate/mist/locations/resources/LocationResource.groovy

```diff
@@ -76,10 +81,26 @@ class LocationResource extends Resource {
     }
 
     @GET
+    @Path("culcenter")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Timed
+    Response getCulCenter() {
+        final List<ServiceLocation> culCenterLocations = culCenterDAO.getCulCenterLocations()
```

Comment:
I get a 500 when calling /culcenter, but the combined endpoint included cultural centers just fine. 

I'm guessing the method call in this line is wrong?